src/react-image-lightbox.js

```diff
@@ -1063,6 +1063,16 @@ class ReactImageLightbox extends Component {
                             </li>
                         </ul>
                     </div>
+
+                    <div // Image caption
+                        className={`caption ril-caption ${styles.caption}`}
```

Comment:
since the `ril-`-less classes were just for backwards compatibility, the `caption` class is not needed here.